app/controllers/your-psp/get.controller.test.js

```diff
@@ -36,10 +38,21 @@ describe('Your psp GET controller', () => {
 
 
   it('should pass the enableStripeOnboardingTaskList to the page ', async () => {
-    process.env.ENABLE_STRIPE_ONBOARDING_TASK_LIST = true
 
     await getController(req, res, next)
-   
-    sinon.assert.calledWith(res.render, `your-psp/index`)    
+    
+    const pageData = res.render.args[0][1]
+    expect(pageData.enableStripeOnboardingTaskList).to.equal(false)
+    
+    sinon.assert.calledWith(res.render, 'your-psp/index')
```

Comment:
Can you please move this to line 44

Reply:
Also we need to set the environment variable here - ENABLE_STRIPE_ONBOARDING_TASK_LIST

Reply:
@AbdirizakIdris you can set the env variable in the unit test directly on the process `process.env.ENABLE_STRIPE_ONBOARDING_TASK_LIST = 'true'` or other values are as required for tests